### DISCUSSION TITLE
How to read the `block` text?

### DISCUSSION THREAD

**Discussion by binodil** [ramnes/notion-sdk-py#194] [General]
Dear All,

# What I did:
I get the `block_id` from helpers.get_id() function. I want to see the content inside the certain block. 
I used `notion.blocks.retrieve(block_id=block_id)`:
# Error:
I believe that I gave all permission for my integration app. 
What went wrong?

`APIResponseError: Could not find block with ID: 5b906cb9-0108-455d-bae4-7fe98b9a64ac. Make sure the relevant pages and databases are shared with your integration.`

**Comment by ramnes** [comment]
Check that your integration appears at the bottom of the menu when you click the ••• button on the top right of your page: <img src="https://user-images.githubusercontent.com/835072/236211710-e31b72f8-68b8-45e2-b505-69cafaec79b0.png" width=200>

It's hard to help without more context.
